debug/index.html

```diff
@@ -58,7 +58,8 @@
     zoom: 12.5,
     center: [-77.01866, 38.888],
     style: 'mapbox://styles/mapbox/streets-v8',
-    hash: true
+    hash: true,
+    maxWorkers: 3
```

Comment:
Right now, this property literally controls the number of spawned workers. If we change the implementation in the future, we can change the property name 😄

Let's go with `workerCount` for an abbreviation-free, grammatically correct, and [best-practice compliant name](http://stackoverflow.com/questions/3742650/naming-conventions-for-number-of-foos-variables).